pkg/scheduler/actions/reclaim/reclaim.go

```diff
@@ -50,7 +50,7 @@ func (ra *Action) Execute(ssn *framework.Session) {
 	klog.V(3).Infof("There are <%d> Jobs and <%d> Queues in total for scheduling.",
 		len(ssn.Jobs), len(ssn.Queues))
 
-	var underRequest []*api.JobInfo
+	// var underRequest []*api.JobInfo
```

Comment:
why remove this?

Reply:
It is useless

Reply:
useless variable has been already deleted